src/forms/FormNav/FormNav.tsx

```diff
@@ -71,9 +71,11 @@ const FormNav = (props: FormNavProps): ReactElement => {
 	 * @param idx
 	 * @param sectionId
 	 */
-	const handleClick = (e, idx: number) => {
+	const handleClick = (e: MouseEvent, idx: number) => {
 		e.preventDefault();
-		sectionsRefs[idx].scrollIntoView({ behavior: "smooth", block: "start", inline: "nearest"});
+		const sectionId = sectionsRefs[idx]?.getAttribute("id");
+		sectionsRefs[idx].scrollIntoView({ behavior: "auto", block: "center", inline: "center"});

```

Comment:
I have several comments here:
1. We still want to keep the behavior as smooth because we want the page to still feel responsive.
2. I believe we can completely remove the inline property, it only affects the horizontal alignment (https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#parameters)
3. I feel the block property (which affects the vertical alignment) needs to be on start. Testing around with the sections example you provided, when clicking on the tabs it takes you to the center of the section, which we don't want because there are forms that might be hundreds of fields long, so it would take them to the middle of the form.

Additionally, testing the sections you provided it seems this doesn't fully solve the issue because if you start clicking one by one, when you click on the last tab it bounces and selects the previous section (section 12).

I'm not sure if this is the best approach but I think we could remove the scroll listener when clicking, and re-add it once the "scrollIntoView"  and the setSelectedTab have finished.